### PR TITLE
Doc reorg based on UX feedback & UI changes.

### PR DESCRIPTION
**Pull request by mcherem**
New order is:
Setup
Config
X12 Settings
Message Types
Endpoints

### CODE REVIEW

File: modules/ROOT/pages/x12-receive-read-settings.adoc [hunk 1]
```diff
@@ -1,167 +1,166 @@
-= X12 Settings: Receive from Partners 
+= Receive from Partners
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
+Refer to these as you set up your X12 settings to receive messages from your partners.
+There are two levels of settings, global and partner.
+Global level settings control all X12 messages received from all partners.
+Partner level settings are optionally configured for each partner and override global settings.
```
Comment (wwarner-salesforce) [inline modules/ROOT/pages/x12-receive-read-settings.adoc:9]: These four lines will currently appear as one paragraph. If you want them to be four paragraphs, add a blank line between each of them.

Comment (mcherem) [reply modules/ROOT/pages/x12-receive-read-settings.adoc:9]: Nope, I think this is fine.

**Review by wwarner-salesforce — APPROVED** [review]

**Comment by wwarner-salesforce** [comment]
I think this is good. Please double check the edits I made to the discussion of endpoint types, and if necessary double check with an SME.

**Comment by wwarner-salesforce** [comment]
Reverted edits for now. Changes to nav look good :)

**Pull request merged**